README.md

```diff
@@ -1,11 +1,10 @@
 # A Nix expression for nightly Rust versions
 
-This is useful for working with the latest Rust nightly (or the nightly from a specific date) on [NixOS].
-Notably, it uses Nix's `builtins.fetchurl` feature to avoid the need to specify source hashes manually.
-This feature is banned in [nixpkgs], so this expression has to exist outside of it.
```

Comment:
That's good then. Yeah on this round of review I was mainly distracted by those linebreaks :P.